src/components/Authentication/AuthModal.tsx

```diff
@@ -0,0 +1,86 @@
+import { useRouter } from "next/router";
+import React, { useState } from "react";
+import { Text, Modal } from "../atomic";
+import Login from "./Login";
+import Signup from "./Signup";
+
+type AuthModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  programName?: string;
+};
+
+const AuthModal = ({ isOpen, onClose, programName = "" }: AuthModalProps) => {
+  const [isLogin, setIsLogin] = useState(false);
+  const router = useRouter();
+
+  return (
+    <Modal isOpen={isOpen} onClose={onClose}>
+      <div className="flex flex-col w-36rem">
+        <button
+          onClick={onClose}
+          className="cursor-pointer focus:outline-none self-end"
+        >
+          <img src="/static/Close.svg" className="h-4 w-4" />
+        </button>
+        {isLogin ? (
+          <div>
+            <div className="flex flex-col space-y-1">
+              {programName ? (
+                <Text h3 b>
+                  Login to join {programName}
```

Comment:
Maybe use a different color to make `programName` more distinguishable?

Reply:
Would italices be enough for distinguishing program name? Not sure what color we could change it to to make it look nice.
![image](https://user-images.githubusercontent.com/20727715/126881122-2c33d8e7-3039-4b3b-abb9-240f8d2be1f2.png)

Reply:
I feel like italics might be too much, but good point. Idk either